data_validation/config_manager.py

```diff
@@ -1177,3 +1177,20 @@ def build_comp_fields(self, col_list: list, exclude_cols: bool = False) -> dict:
         )
 
         return casefold_source_columns
+
+    def default_primary_keys(self) -> list:
```

Comment:
```suggestion
    def auto_list_primary_keys(self) -> list:
```
I think saying default brings a different connotation to the method, like if it would decide by itself which columns would be the PKs 🤔

Reply:
Yep, I agree. Thanks